admin/components/swlogin.js

```diff
@@ -0,0 +1,39 @@
+(function (staticWeb) {
+    "use strict";
+    var Login = function () {
+        if (!(this instanceof Login)) {
+            return new Login();
+        }
+
+        return this.init();
+    }
+    Login.prototype = {
+        createInterface: function () {
+          staticWeb.includeStyle(staticWeb.getAdminPath() + 'css/swadminzone.css');
+          var templateName = 'swlogin-LoggedOut';
```

Comment:
Yes please, naming convention could be good. At least for our own components. Don't know if we need to enforce it, is there any plus on enforcing it?